app/views/templates/project.html.slim

```diff
@@ -0,0 +1,16 @@
+.container-fluid
+  .page-header
+    h1 {{ project | projectName }}
+  .row.overview
+    .col-lg-3.col-md-4.col-xs-12
+      .summary.panel.panel-default ng-show='project.id'
+        .panel-heading.clearfix
+          .label.label-info.pull-right tooltip='Project ID' {{ project.id }}
+          h3.panel-title Details
+        .panel-body
+          ul
```

Comment:
For consistency with other details pages (project list, user details), use a definition list here. The description may be displayed as a `p.lead` element to make it stand out, like in the project list (see `project-panel.html.slim`).